Code/autopkglib/URLDownloader.py

```diff
@@ -125,7 +126,8 @@ def main(self):
         # Download URL.
         url_handle = None
         try:
-            request = urllib2.Request(url=self.env["url"])
+            url = urllib.quote(self.env["url"], 'http://')
```

Comment:
Yes, indeed.

If I omit that the colon ':' will be encoded too:

And I would end up like this:

Couldn't download http://wiresharkdownloads.riverbed.com/wireshark/osx/Wireshark 1.12.3 Intel 64.dmg: unknown url type: http%3A//wiresharkdownloads.riverbed.com/wireshark/osx/Wireshark%201.12.3%20Intel%2064.dmg
Failed.